### DISCUSSION TITLE
v0.1.7 - Easy Launcher

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [DroidWorksStudio/EasyLauncher#84] [Check for Updates]
# What's New

### Implemented Enhancements:

- Added font selection to select font used throughout the launcher - ([e125d44](https://github.com/DroidWorksStudio/EasyLauncher/commit/e125d443bb5635744e76608381e72b3437511e1c))

### Bug Fixes:

- Fixed the update manager to work when reloading app. - ([69dc514](https://github.com/DroidWorksStudio/EasyLauncher/commit/69dc514407dc6d165ae043f04d47b41df8b0f1ec))
- Fixed acra_mail_body string - ([ef4bec6](https://github.com/DroidWorksStudio/EasyLauncher/commit/ef4bec64f9feb75188a6f10b078ccf7317fb20ef))
- Fixed the issue report urls for the wiki. - ([9a1fe83](https://github.com/DroidWorksStudio/EasyLauncher/commit/9a1fe83330f7797133769513370562d270903f9c))

### Dependency Updates:

- Updated the dependencies for the launcher. - ([cb91acf](https://github.com/DroidWorksStudio/EasyLauncher/commit/cb91acf50855c7f5c0ae56a68bf012d5f27e5ed2))

### Language Support:

- Updated Language Files. ([#81](https://github.com/DroidWorksStudio/EasyLauncher/issues/81)) - ([9d825fa](https://github.com/DroidWorksStudio/EasyLauncher/commit/9d825fa3c992d245000299f610b19f9280dd0ee7))
- Updated Language Files. ([#78](https://github.com/DroidWorksStudio/EasyLauncher/issues/78)) - ([2d743f8](https://github.com/DroidWorksStudio/EasyLauncher/commit/2d743f8706cc1cf47e0c20d625f183c92f344027))

### Releases:

- Prepare for version 0.1.7 - ([3e1dd98](https://github.com/DroidWorksStudio/EasyLauncher/commit/3e1dd982254444f2e55551f475c2a3056f13b05c))

**Full Changelog**: https://github.com/DroidWorksStudio/EasyLauncher/compare/0.1.6...0.1.7

<hr /><em>This discussion was created from the release <a href='https://github.com/DroidWorksStudio/EasyLauncher/releases/tag/0.1.7'>v0.1.7 - Easy Launcher</a>.</em>

**Comment by due3die** [comment]
It was crashing always not able to use the Launcher.

**Reply by CreativeCodeCat** [reply]
If you are crashing I wouldn't know unless you send the crash report via the email.

**Reply by due3die** [reply]
What is the email id?

![Screenshot_2024-07-06-20-57-19-93_1d151929efb2ddc7073e99a7f966a46b.jpg](https://github.com/DroidWorksStudio/EasyLauncher/assets/3252373/3e9d40c7-ca54-4761-bb37-eee9b20682a8)

**Reply by due3die** [reply]
> If you are crashing I wouldn't know unless you send the crash report via the email.

I have already sent an email, kindly check and confirm.

**Reply by CreativeCodeCat** [reply]
ok i got the email but the information that is requested in the email wasnt filled in. ill check the log now though. some reason your email didnt like my gmail :)

**Reply by CreativeCodeCat** [reply]
> I have already sent an email, kindly check and confirm.

which build are you using internet or no internet?

**Reply by CreativeCodeCat** [reply]
@due3die I found the issue and will be fixed in v0.1.8.

**Reply by due3die** [reply]
> > I have already sent an email, kindly check and confirm.
> 
> which build are you using internet or no internet?

Dev, No internet version.

**Reply by due3die** [reply]
> @due3die I found the issue and will be fixed in v0.1.8.

Okay. Waiting developer.

**Reply by due3die** [reply]
> using

What is the difference between the two versions?

**Reply by due3die** [reply]
> @due3die I found the issue and will be fixed in v0.1.8.

Got it thanks. It's working dev.

**Reply by CreativeCodeCat** [reply]
> What is the difference between the two versions?

One allows internet access and one doesn't. So the one without internet can not access the widgets that require internet like the weather for example.
